src/layouts/components/Header.tsx

```diff
@@ -237,3 +323,145 @@ function HeaderLink(props: {
+        testId={`${viewPort}.HeaderLink.Auctions`}
+      />
+      <HeaderLink
+        className="flex justify-start border-b border-[#E5E5E5] dark:border-gray-700 px-4 p-1.5"
+        text="Oracles"
+        pathname="/oracles"
+        testId={`${viewPort}.HeaderLink.Oracles`}
+      />
+      <HeaderLink
+        className="flex justify-start border-b border-[#E5E5E5] dark:border-gray-700 px-4 p-1.5"
+        text="Tokens"
+        pathname="/tokens"
+        testId={`${viewPort}.HeaderLink.Tokens`}
+      />
+      <HeaderLink
+        className="flex justify-start border-b border-[#E5E5E5] dark:border-gray-700 px-4 p-1.5"
+        text="Masternodes"
+        pathname="/masternodes"
+        testId={`${viewPort}.HeaderLink.Masternodes`}
+      />
+      {/* <HeaderLink
+        className="flex justify-start border-b border-gray-100 dark:border-gray-700 p-1.5 md:px-4"
+        text="Consortium"
+        pathname="/consortium/asset_breakdown"
+        testId={`${viewPort}.HeaderLink.Consortium`}
+      /> */}
+      <HeaderLink
+        className="flex justify-start border-b border-[#E5E5E5] dark:border-gray-700 px-4 p-1.5"
+        text="On-Chain Governance"
+        pathname="/on-chain-governance"
+        testId={`${viewPort}.HeaderLink.Consortium`}
+      />
+    </div>
+  );
+}
+
+function MoreDropdown(): JSX.Element {
+  const [isItemClicked, setIsItemClicked] = useState(false);
+  const router = useRouter();
+  useEffect(() => {
+    setIsItemClicked(
+      dropDownLinks.some((ddl) =>
+        router.pathname.includes(ddl.rootPathName.toLowerCase())
+      )
+    );
+  }, [router.pathname]);
+
+  return (
+    <Menu as="div" className="relative">
+      {({ open }) => (
+        <>
+          <Menu.Button
+            className={classNames(
+              "flex flex-row items-center mx-4 dark:hover:text-dark-50 cursor-pointer text-lg hover:text-primary-500",
+              {
+                "dark:text-dark-50 text-primary-500": isItemClicked,
+              }
+            )}
+          >
+            More
+            {open ? (
+              <IoChevronUp className="ml-2" size={20} />
+            ) : (
+              <IoChevronDown className="ml-2" size={20} />
+            )}
+          </Menu.Button>
+          <Transition
+            enter="transition duration-100 ease-out"
+            enterFrom="transform scale-95 opacity-0"
+            enterTo="transform scale-100 opacity-100"
+            leave="transition duration-75 ease-out"
+            leaveFrom="transform scale-100 opacity-100"
+            leaveTo="transform scale-95 opacity-0"
+          >
+            <Menu.Items className="absolute m-4 min-w-max flex flex-col divide-y bg-white border rounded-lg border-gray-200 dark:border-gray-700 dark:bg-gray-700">
+              {dropDownLinks.map((item, index) => {
+                return (
+                  <Menu.Item key={index}>
+                    <>
+                      <Link href={{ pathname: item.link }} passHref>
```

Comment:
can you check how to close the `More` dropdown after user click on `On-Chain Governance`?